src/main/java/core/mate/academy/model/ExcavatorProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer {
+    @Override
+    public List<Object> get() {
+        return new ArrayList<>((Collection) new Excavator());
```

Comment:
You are trying to cast an Excavator object to a Collection, which is incorrect and will result in a ClassCastException. Instead, you should create a List<Excavator> and add new Excavator objects to it.